src/operator/OrgOverlay.tsx

```diff
@@ -339,6 +339,28 @@ export const OrgOverlay: FC = () => {
                       </Grid.Column>
                     </Grid.Row>
                   )}
+                  {limits?.ioxQuery && (
```

Comment:
Seems like [ioxQuery](https://github.com/influxdata/openapi/blob/3380e8851d26ca33131a3e183d7147b42fdf2a09/contracts/priv/unity.yml#L2738) is defined under the type [OrgLimits](https://github.com/influxdata/openapi/blob/35ecbc2e2890e77756d0be0f801b9e503a8ffd44/contracts/priv/quartz-oem.yml#L471), but `limits` is a type of [OperatorOrgLimits](https://github.com/influxdata/openapi/blob/3380e8851d26ca33131a3e183d7147b42fdf2a09/contracts/priv/unity.yml#L1890)

Reply:
I think this is now resolved after https://github.com/influxdata/openapi/pull/649